x-pack/plugins/security_solution/public/management/components/response_action_file_download_link/response_action_file_download_link.tsx

```diff
@@ -97,31 +109,32 @@ export const ResponseActionFileDownloadLink = memo<ResponseActionFileDownloadLin
     }
 
     return (
-      <>
-        <EuiButtonEmpty
-          href={downloadUrl}
-          iconType="download"
-          data-test-subj={getTestId('downloadButton')}
-          flush="left"
-          style={STYLE_INHERIT_FONT_FAMILY}
-          download
-        >
-          <EuiText size="s">{buttonTitle}</EuiText>
-        </EuiButtonEmpty>
-        <EuiText
-          size="s"
-          className="eui-displayInline"
-          data-test-subj={getTestId('passcodeMessage')}
-        >
-          <FormattedMessage
-            id="xpack.securitySolution.responseActionFileDownloadLink.passcodeInfo"
-            defaultMessage="(ZIP file passcode: {passcode})"
-            values={{
-              passcode: 'elastic',
-            }}
-          />
-        </EuiText>
-      </>
+      <EuiFlexGroup alignItems="center" gutterSize="none">
```

Comment:
can you add `data-test-subj={dataTestSubj}` here so that the entire component gets a test subject?

Reply:
72aed7e3c97db58552342377db11fd361b4ffaa7